src/app/gateway/gateway-overview/gateway-overview.component.ts

```diff
@@ -65,9 +66,12 @@ export class GatewayOverviewComponent implements OnInit, OnChanges, OnDestroy {
     this.canEdit = this.meService.hasAccessToTargetOrganization(
       OrganizationAccessScope.GatewayWrite
     );
-    if (this.router.url === '/gateways') {
-      this.router.navigateByUrl('/gateways/list', { replaceUrl: true });
-    }
+    // Subscribe to route change to root and route to list view
+    this.router.events.subscribe((e) => {
```

Comment:
There is some error with this logic. First time enter on "gateways" shows nothing and redirect away from gateway doesn't work.